deployment-examples/docker-compose/multi-worker-with-grace-period.json5

```diff
@@ -0,0 +1,134 @@
+// Multi-worker configuration with eviction grace period enabled
+// This prevents premature eviction of recently accessed items, protecting
+// them from being evicted while still needed by in-flight actions.
+{
+  stores: [
+    {
+      name: "CAS_FS",
+      filesystem: {
+        content_path: "/tmp/nativelink-multi-test/cas/content",
+        temp_path: "/tmp/nativelink-multi-test/cas/temp",
+        eviction_policy: {
+          max_bytes: 10000000000, // 10GB
```

Comment:
Nit: We should be able to use `10GB` directly (and similar for other units here) not as a comment

Reply:
pre-commit doesn't allow for the 10GB here but the API does.